src/dialog-user/components/dialog-user/dialogField.ts

```diff
@@ -95,6 +98,19 @@ export class DialogFieldController extends DialogFieldClass {
   public refreshSingleField() {
     this.singleRefresh({ field: this.field.name });
   }
+
+  /**
+   * This method is setting the default_value for a tag control's select box.
+   * In case the default_value is not set for the ng-model of the component,
+   * an empty value option is displayed
+   * @memberof DialogFieldController
+   * @function setDefaultValue
+   */
+  private setDefaultValue() {
+    let defaultOption = _.find(this.dialogField.values, { id: null })
+    defaultOption.id = 0;
```

Comment:
What if there is no defaultOption? I guess leaving blank space as was before is good isn't it?

Reply:
@karelhala updated. Thanks :+1: